RULES.md

```diff
@@ -262,6 +264,15 @@ Empty csv file found in the archive: file does not have any headers, or is a req
 ##### References:
 * [GTFS files requirements](https://gtfs.org/reference/static#file-requirements)
 
+#### EqualShapeDistanceDiffCoordinatesNotice
+
+<a name="EqualShapeDistanceDiffCoordinatesNotice"/>
+
+When sorted by `shape.shape_pt_sequence`, two consecutive shape points with different coordinates must not have equal values for `shape_dist_traveled`.
```

Comment:
In order to give the user a bit more context, I would rephrase this to something like:
When sorted by `shape.shape_pt_sequence`, the values for `shape_dist_traveled` must increase. 
Two consecutive points with equal values for `shape_dist_traveled` and different lat / lon coordinates indicate an error.